src/main/java/witchinggadgets/common/items/armor/ItemPrimordialArmor.java

```diff
@@ -147,19 +152,15 @@ public void onArmorTick(World world, EntityPlayer player, ItemStack stack) {
         }
 
         if (this.armorType == 3) {
-            if (!player.capabilities.isFlying && player.moveForward > 0.0F) {
```

Comment:
I'm not sure if the new code would translate 1:1 with the intended, so it may result in this portion of the gameplay not correctly functioning